cypress/test/campaignTests/Banner.cy.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { mount } from '@cypress/react';
+import Banner from '../../../components/campaigns/Banner';
+
+describe('Banner Component', () => {
+  it('should render the banner when the date is within the valid range', () => {
+    const today = new Date();
+    const day = today.getUTCDate();
+    const month = today.getUTCMonth();
+    const year = today.getUTCFullYear();
+
+    if (year > 2022 || month !== 10 || day < 6) {
+      mount(<Banner />);
+      cy.get('.bg-gray-100').should('not.exist');
+ 
+    } else {
+      mount(<Banner />);
+     // cy.wait(1000); // Adjust the wait time as needed
+     cy.get('.bg-gray-100').should('be.visible');
+    
+    }
+  });
+
+  it('should display the correct message when the date is within the valid range', () => {
+    const today = new Date();
+    const day = today.getUTCDate();
+    const month = today.getUTCMonth();
+    const year = today.getUTCFullYear();
+
+    if (year > 2022 || month !== 10 || day < 6) {
+      mount(<Banner />);
+      cy.contains('.font-medium', 'AsyncAPI Conference 2022 has ended').should('not.exist');
+    } else {
+      mount(<Banner />);
+     // cy.wait(1000); // Adjust the wait time as needed
+      cy.contains('.font-medium', 'AsyncAPI Conference 2022 has ended').should('be.visible');
+    }
+  });
+
+  it('should have a link to the recordings playlist', () => {
+    const today = new Date();
+    const day = today.getUTCDate();
+    const month = today.getUTCMonth();
+    const year = today.getUTCFullYear();
+
+    if (year === 2022 && month === 10 && day >= 6) {
+      mount(<Banner />);
+     //cy.wait(1000); // Adjust the wait time as needed
```

Comment:
Why the wait time is included in the tests?

Reply:
Included the wait time earlier , had to test something specific , removed it , though I will remove the comments .

Reply:
@akshatnema made a new commit removing the wait time , I am sorry for the commit message here , VS code did it , I was not able to connect to git , so had to push through source control in vs .